src/mod_private_rdbms.erl

```diff
@@ -40,47 +40,63 @@
 -include("mongoose.hrl").
 -include("jlib.hrl").
 
-init(_Host, _Opts) ->
+init(Host, _Opts) ->
+    mongoose_rdbms:prepare(private_select_data, private_storage,
+                           [username, namespace],
+                           <<"SELECT data FROM private_storage WHERE username=? AND namespace=?">>),
+    mongoose_rdbms:prepare(private_select_namespaces, private_storage,
+                           [username],
+                           <<"SELECT namespace FROM private_storage WHERE username=?">>),
+    mongoose_rdbms:prepare(private_remove_user, private_storage,
+                           [username],
+                           <<"DELETE FROM private_storage WHERE username=?">>),
+    rdbms_queries:prepare_upsert(Host, private_upsert, private_storage,
+                                 [<<"username">>, <<"namespace">>, <<"data">>],
+                                 [<<"data">>],
+                                 [<<"username">>, <<"namespace">>]),
```

Comment:
For the future: Could we estimate the number of prepared queries if we extend this to all modules and the impact on connection setup time introduced by preparing all of them? We could just double check that there is no significant memory impact as well (probably not).

Reply:
Queries are prepared on demand first time when executed with that connection. 
I think preparing queries would take similar amount of time as just making a query without parameters.

Memory is a good thing to check ;)

Reply:
Ok, so it's already done in the "lazy" way, that's good. Memory should not explode, it's just good to check.